.github/workflows/Tests.yaml

```diff
@@ -714,7 +717,7 @@ ${{ matrix.build_type }}-pch-${{ matrix.use_pch || 'ON' }}"
         if: matrix.build_type == 'Release'
         working-directory: build
         run: |
-          make -j2 Benchmark
+          make -j${NUMBER_OF_CORES} Benchmark
```

Comment:
The "Run non-unit tests" step below still uses 2 cores. Switch to 1 core and/or add a comment why it's not the max?

Reply:
I've added a comment. That was intentional because I've had them time out sooner than the unit tests on my desktop. Having said that, we could try 4 cores and increase timeouts. That might be better?